src/plugins/vis_type_timeline/server/plugin.ts

```diff
@@ -173,12 +173,9 @@ export class Plugin {
         }),
         value: config.graphiteUrls && config.graphiteUrls.length ? config.graphiteUrls[0] : null,
         description: i18n.translate('timeline.uiSettings.graphiteURLDescription', {
-          defaultMessage:
-            '{experimentalLabel} The <a href="https://www.hostedgraphite.com/UID/ACCESS_KEY/graphite" target="_blank" rel="noopener">URL</a> of your graphite host',
+          defaultMessage: '{experimentalLabel} The URL of your graphite host',
           values: { experimentalLabel: `<em>[${experimentalLabel}]</em>` },
         }),
-        type: 'select',
```

Comment:
Non-blocking: This might be a regression if customer is using allow list. 

They could choose from a 'select' before. And they have to manually input the whole string of url address after.

Reply:
Text box is enough for now. Customer needs to manually input.